pyproject.toml

```diff
@@ -147,7 +147,7 @@ markers = [
     'needs_vtk_version(version): skip test unless VTK version is at least as specified.',
     'needs_download: this test downloads data during execution',
 ]
-
+addopts = '--image_cache_dir ./tests/plotting/image_cache'
```

Comment:
This shouldn't be needed if `main` is merged in. Was handled in #3748

Reply:
We need to merge it in as there was also a bug with marks for skipping tests and high variance leaking out.

Reply:
As far as I can tell, both options are equivalent. I merged anyway, but I think we should revert that line change since I prefer `image_cache_dir` over `addopts`.